cluster-autoscaler/cloudprovider/gce/gce_cloud_provider.go

```diff
@@ -200,6 +200,12 @@ func (gce *GceCloudProvider) Refresh() error {
 	return gce.gceManager.Refresh()
 }
 
+// GetProjectInfo returns the project id, location and cluster name.
+func (gce *GceCloudProvider) GetProjectInfo() (projectId, location, clusterName string) {
+	manager := gce.gceManager
```

Comment:
You don't need to create local variable.

Reply:
I don't think it hurts to create it and have a shorter return line. I'm fine either way.